plutus-pab/src/Cardano/Node/RandomTx.hs

```diff
@@ -78,10 +80,9 @@ runGenRandomTx =
      implementation. Please make sure to read it's documentation if you want to split
      the value into more than 10 outputs.
 -}
-generateTx :: GenIO -> UtxoIndex -> IO Tx
-generateTx gen (UtxoIndex utxo) = do
-  (sourcePrivKey, sourcePubKey) <- pickNEL gen keyPairs
-  (_, targetPubKey) <- pickNEL gen keyPairs
+generateTx :: GenIO -> Slot -> UtxoIndex -> IO Tx
+generateTx gen slot (UtxoIndex utxo) = do
```

Comment:
Haddocks for the argument (the slot is the current slot, right?)

Reply:
I will update the haddock.